### ISSUE TITLE
Upload does not work

### ISSUE THREAD

**Issue by Jozzeh** [icetee/remote-ftp#282]
( Latest version of atom and remote-ftp , tested on Windows 7 and OS X 10.9.5 , with ftp and sftp config file )

**Bug**: When right-clicking a file in the local tree-view and choosing 'Upload' or 'Sync Local->Remote', the file/folder is not uploaded. No error is thrown.

When opening a new file locally and saving it, it gets uploaded. (normal behavior) 
Download from remote to local does work. (right click in remote tree, and choosing download)

**Comment by Jozzeh** [comment]
After some testing I have solved this issue. 

As said in the issue, this bug is seen when right clicking in the local tree-view on a file or folder and choosing 'Upload'. 

In the file lib/remote-ftp.js in the 'remote-ftp:upload-selected' function on line 298 it says:
`var locals = $('.ftptree-view .selected').map(function () { ..........`

This means it will take the selected item in the ftptree-view and upload it. However, the upload option is available on the local tree view. So i've changed .ftptree-view to .tree-view and then the upload works again.

In short ( lib/remote-ftp.js ):
Line 298 : `var locals = $('.ftptree-view .selected').map(function () {`
Change to 
Line 298 : `var locals = $('.tree-view .selected').map(function () {`

and for Sync Local -> Remote
Line 344 : `var locals = $('.ftptree-view .selected').map(function () {`
Change to 
Line 344 : `var locals = $('.tree-view .selected').map(function () {`

**Comment by james-jlo-long** [comment]
This was really bugging me yesterday. Thanks for posting a fix.

**Comment by webcraftniray** [comment]
@Jozzeh Many thanks for this fix... Works a treat... :+1:

**Comment by YarmoM** [comment]
Will this be fixed in an upcoming release?

**Comment by mdurchholz** [comment]
@Jozzeh Nice catch, thanks for the fix!

**Comment by jpxd** [comment]
Should be resolved in the latest release
